### PR TITLE
Lab3 html/css

### PR DESCRIPTION
**Pull request by SoulHowl**
Barsukevich S. 953504
Lab3 pull request

### CODE REVIEW

**Review by katetushkan — CHANGES_REQUESTED** [review]
deploy link for html and css?

**Comment by SoulHowl** [comment]
this should work
https://soulhowl.github.io

File: Lab3/Weather/index.html [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+<section class="wrapper-main">
+    <section class="upper">
+        <section class="temperature">
+
+            <article class="widget">
+                <div class="time">
+                    <h6>12:12</h6></div>
+                <div class="weatherIcon"></div>
+                <div class="weatherData">
+                    <h1 class="temperature-1">25&deg;</h1>
```
Comment (katetushkan) [inline Lab3/Weather/index.html]: U can't use `<h1>` after `<h6>`

File: Lab3/Weather/index.html [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+            </li>
+        </ul>
+    </nav>
+</header>
+<section class="wrapper-main">
+    <section class="upper">
+        <section class="temperature">
+
+            <article class="widget">
+                <div class="time">
```
Comment (katetushkan) [inline Lab3/Weather/index.html]: use tag `<time>`

File: Lab3/Weather/index.html [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+                    <h6>12:12</h6></div>
+                <div class="weatherIcon"></div>
+                <div class="weatherData">
+                    <h1 class="temperature-1">25&deg;</h1>
+                    <h2 class="description">Partly Cloudy</h2>
```
Comment (katetushkan) [inline Lab3/Weather/index.html]: tags like `<h1..6>` should be used only like headers for page or a section. Don't use them to represent a set of information. Use `<p>`

File: Lab3/Weather/index.html [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+        </section>
+    </section>
+    <section class="lower">
+        <section class="humidity">
+            <div class="circular">
+                <div class="inner">
+                </div>
+                <div class="outer">
+                </div>
+                <div class="numb">
```
Comment (katetushkan) [inline Lab3/Weather/index.html]: should be `<p>`

File: Lab3/Weather/index.html [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+                    <div class="bar right">
+                        <div class="progress">
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="pressure">
+            <div class="with-digits">
+                <div class="upper-digit"><h3 class="upper-d">18</h3></div>
```
Comment (katetushkan) [inline Lab3/Weather/index.html]: should be `<p>`

File: Lab3/Weather/Map.html [hunk 6]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```
Comment (katetushkan) [inline Lab3/Weather/Map.html:1]: there is no mobile version here (
Screenshot 2022-04-05 at 14 12 19
also, smth is wrong with nav 
Screenshot 2022-04-05 at 14 12 43

**Review by katetushkan — CHANGES_REQUESTED** [review]

**Comment by SoulHowl** [comment]
Proj update: fixed small issues, fixed design for phones

**Review by katetushkan — APPROVED** [review]
It's `7`. U OK with that or wanna upgrade?

**Comment by SoulHowl** [comment]
@katetushkan , it's ok.

**Pull request merged**